packages/template-retail-react-app/app/constants.js

```diff
@@ -201,3 +201,19 @@ export const STORE_LOCATOR_DISTANCE = 100
 export const STORE_LOCATOR_NUM_STORES_PER_LOAD = 10
 export const STORE_LOCATOR_DISTANCE_UNIT = 'km'
 export const STORE_LOCATOR_IS_ENABLED = true
+
+// Constants for shopper context
+export const SHOPPER_CONTEXT_SEARCH_PARAMS = {
+    SOURCE_CODE: 'sourceCode',
+    EFFECTIVE_DATE_TIME: 'effectiveDateTime',
+    CUSTOMER_GROUP_IDS: 'customerGroupIds'
+}
+export const SHOPPER_CONTEXT_CUSTOM_QUALIFIERS_SEARCH_PARAMS = {
+    DEVICE_TYPE: 'deviceType',
+    IP_ADDRESS: 'ipAddress',
+    OPERATING_SYSTEM: 'operatingSystem'
+}
+export const SHOPPER_CONTEXT_ASSIGNMENT_QUALIFIERS_SEARCH_PARAMS = {
```

Comment:
these values are fields being set in BM, it is not a constant for all sandbox. We don't need to set it here.